examples/siro_sandbox/sandbox_app.py

```diff
@@ -538,14 +539,12 @@ def _save_recorded_keyframes_to_file(self):
     def _update_cursor_style(self, post_sim_update_dict):
         do_update_cursor = False
         if self._cursor_style is None:
-            if self._first_person_mode:
-                self._cursor_style = Application.Cursor.HIDDEN_LOCKED
-            else:
-                self._cursor_style = Application.Cursor.ARROW
+            self._cursor_style = Application.Cursor.ARROW
             do_update_cursor = True
         else:
-            if self._first_person_mode and self.gui_input.get_mouse_button(
-                GuiInput.MouseNS.LEFT
+            if (
+                self._first_person_mode
+                and self.gui_input.get_mouse_button_down(GuiInput.MouseNS.LEFT)
```

Comment:
Note change to use `get_mouse_button_down`. This is only True on the frame you press down the button. `get_mouse_button` is True every frame as long as you keep holding the button, which we don't want here.